### ISSUE TITLE
CKEditor does not allow block level link

### ISSUE THREAD

**Issue by brharp** [ccswbs/hjckrrh#646]
I would like to wrap a few block tags in a link (valid in HTML5), but CKEditor rewrites the code such that the links are placed inside block tags.

For example, this:
```
<a href="/ccs/infosec/resources">
  <figure class="thumbnail">
    <img src="resources.jpg">
    <figcaption>Resources</figcaption>
  </figure>
</a>
```
is turned into this:
```
<figure class="thumbnail">
  <a href="/ccs/infosec/resources">
    <img src="resources.jpg" />
  </a>
  <figcaption>
    <a href="/ccs/infosec/resources">Resources</a>
  </figcaption>
</figure>
```
Which won't do because adjacent links are not accessible.

This is a [known](http://dev.ckeditor.com/ticket/7961) [issue](http://dev.ckeditor.com/ticket/9457) in CKEditor, unlikely to be fixed anytime soon.

See also [this question on SO](http://stackoverflow.com/questions/19825802/how-to-configure-ckeditor-to-allow-html-block-level-tags-to-be-wrapped-in-an-anc)

**Comment by tqureshi-uog** [comment]
If CKeditor ever releases a fix for this, we can re-open this issue.
